src/Graphics/OpenGLContext/GLSL/glsl_CombinerProgramUniformFactory.cpp

```diff
@@ -242,7 +242,7 @@ class URasterInfo : public UniformGroup {
 		float texCoordOffset[2][2] = { 0.0f, 0.0f };
 		if (isTexRect && !isNativeRes) {
 			float scale[2] = { 0.0f, 0.0f };
-			if (config.graphics2D.enableNativeResTexrects != 0) {
+			if (config.graphics2D.enableNativeResTexrects != 0 && gDP.otherMode.textureFilter == G_TF_BILERP) {
```

Comment:
GraphicsDrawer::drawTexturedRect uses
const bool bUseBilinear = gDP.otherMode.textureFilter != 0;
0 is G_TF_POINT
I think it is better to use that approach here as well:
`if (config.graphics2D.enableNativeResTexrects != 0 && gDP.otherMode.textureFilter != G_TF_POINT) {
`

Reply:
Done